pages/about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>About - Esport Events</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+        <link rel="stylesheet" href="../assets/css/index.css">
+
+    </head>
+
+<body>
+    <nav id="navbar-desk" class="nav-desk isDesktop">
+        <div class="dark-bar isDesktop">
+            <div class="bar-content">
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Login</a>
+            </div>
+        </div>
+        <div class="desktop-nav isDesktop">
+            <a href="/index.html" class="head-logo-link">
+                <div class="head-logo">
+                    <i class="fa-solid fa-gamepad fa-2xl logo-icon"></i>
+                    <h5 class="title">E-Sport <br> Events</h5>
+                </div>
+            </a>
+            <ul class="nav-links-desk">
+                <li><a class="orangecolor" href="/index.html">Home</a></li>
+                <li><a href="/index.html#main-program">Program</a></li>
+                <li><a href="/index.html#FeaturedSpeakers">Speakers</a></li>
+                <li><a href="/index.html#sponsors">Partners</a></li>
+            </ul>
+            <a class="main-btn" href="#">EE Compaign</a>
+        </div>
+    </nav>
+    <header class="about-head-cont">
+        <div class="mobile-menu-container hide isMobile">
+            <div class="close-icon" id="close-mobile-menu">
+                <i class="fa-solid fa-xmark fa-xl"></i>
+            </div>
+            <div class="mobile-menu">
+
+                <ul class="mobile-links">
+                    <li class="mobile-link"><a href="/index.html" class="title-link">Home</a></li>
+                    <li class="mobile-link"><a href="/index.html#main-program" class="title-link">Main Program</a></li>
+                    <li class="mobile-link"><a href="/index.html#FeaturedSpeakers" class="title-link">Speakers</a></li>
+                    <li class="mobile-link"><a href="/index.html#sponsors" class="title-link">Partners</a></li>
+                    <li class="mobile-link"><a href="/pages/about.html" class="title-link">About</a></li>
+                </ul>
```

Comment:
[OPTIONAL] Kindly try to open the mobile menu on the about page as you did on the home page because now there is no navigation to come back to the home page in the mobile version.